docs/routing/special_routes.md

```diff
@@ -0,0 +1,62 @@
+# Special routes
+
+To see special routes, use the `Show special` switcher at the top of the routes page
+
+It is used to render errors, if an application with the `Primary` or `Essential` type falls on the page, ILC will not find a match between the address bar and `Route`, `Template` will not be specified in the route chain, etc., ILC will use `Special route` or simple template with an error, depending on the situation:
+
+## '404' Error
+
+![ILC registry special routes switcher](../assets/routes/special-routes-switcher.png)
+
+- This error usually means that the requested resource was not found on the server.
+
+- When using `Special route` all standard routes are ignored, including `*` with any `Order pos`.
+
+- If you want something else to be displayed on the page besides the error (header, footer, etc.), you need to add them to the [Slot](./route_configuration_options.md#slot-configuration) as in the screenshot below (This is necessary because standard routes are ignored).
```

Comment:
oh, you have additional information here regarding `ignorring`. it's better to combine this two points to one, like I did